leshan-server-cf/src/test/java/org/eclipse/leshan/server/californium/LeshanServerTest.java

```diff
@@ -81,11 +81,34 @@ public void testStartStopDestroy() throws InterruptedException {
         assertEquals("All news created threads must be destroyed", numberOfThreadbefore, Thread.activeCount());
     }
 
+    @Test
+    public void testStartStopDestroyQueueModeDisabled() throws InterruptedException {
+        // look at nb active thread before.
+        int numberOfThreadbefore = Thread.activeCount();
+
+        LeshanServer server = new LeshanServerBuilder().setLocalAddress(new InetSocketAddress(0))
+                .disableQueueModeSupport().build();
+        server.start();
+        Thread.sleep(100);
+        // HACK force creation thread creation.
+        forceThreadsCreation(server);
+        Thread.sleep(100);
+        server.stop();
+        Thread.sleep(100);
+        server.destroy();
+
+        // ensure all thread are destroyed
+        Thread.sleep(500);
+        assertEquals("All news created threads must be destroyed", numberOfThreadbefore, Thread.activeCount());
```

Comment:
My first idea would have been to not check all the counting thread part (just a simple create/start/stop/destroy), but thinking a bit more about it I don't know what is the best way :shrug: 

I guess we can keep it like this :thinking:, any opinion ?

Reply:
I just copied the other tests to keep the same style, and all the other tests are checking threads, so I thought it was something to check in this specific config. Anyway, I don't have a strong opinion on that since I didn't write those tests :smiley:

Reply:
>  all the other tests are checking threads

Almost all, the first one just checks just start/stop/start without thread counting.

I don't find any really good reason to remove or keep it. So I guess we can let it like this :sweat_smile: